code/controllers/subsystem/blackbox.dm

```diff
@@ -32,10 +32,17 @@ SUBSYSTEM_DEF(blackbox)
 
 	CheckPlayerCount()
 
-	if(CONFIG_GET(flag/use_exp_tracking))
-		if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
+	if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
```

Comment:
is there a point to flipping these two?

Reply:
ah i see, makes sense

Reply:
I flipped them because it was the config that prevented metacoins from paying out. Now, they should pay out regardless of the job experience config

Reply:
I flipped them because it was the config that prevented metacoins from paying out. Now, they should pay out regardless of the job experience config